mlflow/sagemaker/__init__.py

```diff
@@ -13,7 +13,7 @@
 from mlflow.utils.logging_utils import eprint
 from mlflow.utils.file_utils import TempDir, _copy_project
 
-DEFAULT_IMAGE_NAME = "mlflow_sage"
+DEFAULT_IMAGE_NAME = "mlflow-pyfunc"
```

Comment:
would you consider this a breaking change?

Reply:
I think it's fine.